docs/site/FAQ.md

```diff
@@ -17,16 +17,11 @@ summary: LoopBack 4 is a completely new framework, also known as LoopBack-Next.
 
 See [Crafting LoopBack 4](Crafting-LoopBack-4.md) for more details.
 
-### What’s the timeline for LoopBack 4?
-
-See
-[Upcoming releases](https://github.com/strongloop/loopback-next/wiki/Upcoming-Releases).
-
 ### Where are the tutorials?
 
 See [Examples and tutorials](Examples-and-tutorials.md).
 
-### What features are planned ?
+### What features are planned?
```

Comment:
Should we split this section in two parts - what has been done and what is being planned? We can also point readers to "LB3 Feature Parity" issue.

Reply:
Actually, I was thinking whether it's better to remove this list completely because features in plan are dynamic and changes over time.  Maybe it's better to put something like: 
Our roadmap is captured in <link to github search with "roadmap" label>.  You can also check out our monthly milestone plans: https://github.com/strongloop/loopback-next/labels/Monthly%20Milestone.  

For LoopBack 3 users, here is the list of LB3 Feature parity we plan to implement in the future: https://github.com/strongloop/loopback-next/issues/1920.

What do you think?

Reply:
Your proposal looks good 👍